src/azure-cli/azure/cli/command_modules/apim/tests/latest/policy.xml

```diff
@@ -0,0 +1,6 @@
+<http-data-source>
+	<http-request>
+		<set-method>GET</set-method>
+		<set-url>https://some.service.com</set-url>
+	</http-request>
+</http-data-source>
```

Comment:
Could you please move this file to `src/azure-cli/azure/cli/command_modules/apim/tests/latest/data` folder, this will help us manage our packaging tests

Reply:
I'm skipping I'm afraid if I changed little what other test cases might break, I will create a work item and take it up in next iteration so for now let's keep in same folder.